.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -105,14 +105,16 @@ jobs:
       - name: Create role for AWS access from the sample app
         id: create_service_account
         run: |
-          eksctl create iamserviceaccount \
+          source ${{ env.TEST_RESOURCES_FOLDER }}/.github/workflows/util/execute_and_retry.sh
```

Comment:
Is there a better way to call this file? We can turn it into an action that is reusable.

Reply:
Done